src/motion-queue.h

```diff
@@ -90,6 +104,11 @@ class MotionQueue {
 
   // Shutdown. If !flush_queue: immediate, even if motors are still moving.
   virtual void Shutdown(bool flush_queue) = 0;
+
+  //Return the pointer to the status register.
+  virtual const struct QueueStatus get_status() = 0;
```

Comment:
I prefer to have 'cheap' methods be lower-case, everything else Uppercase. GetStatus() maybe ?